portfolio/src/main/webapp/index.html

```diff
@@ -39,6 +39,9 @@
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#interests">Interests</a></li>
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#comments">Comments</a></li>
+          <br>
```

Comment:
abusing `<br>` to create padding is an anti-pattern. I don't believe it's valid for `<br>` to be a direct child of `<ul>`. Instead, consider adding `margin-top: 16px` to `#login` or some other CSS-way (you can tag the `<li>` with a class/id too).